Header header
Header image_header
BoundingBox[] bounding_boxes
int32 objNum
